wordpress-mobile/WordPress-iOS
# Notification Entities + Swift Mk 2: Blocks + Groups

#### Details:

Migrating **NotificationBlock** + **NotificationBlockGroup**. Reference #5613 

I've initially (also) migrated NotificationRange + NotificationMedia, but... decided to split that in two PR's, to ease review.

Needs Review: @kwonye 
Thanks in advance Will!!


#### Testing:
- Verify that the Notifications List works as expected
- Verify that the Details Views for **[Comments, Likes, Follow, Badges]** work as expected
